frontend/src/workspace.rs

```diff
@@ -452,7 +452,7 @@ impl Component for Workspace {
 }
 
 impl Workspace {
-    pub fn register_terminals(&mut self, id: ModuleId, inputs: &[LineType], outputs: &[LineType]) {
+    pub fn register_terminals(&mut self, id: ModuleId, inputs: &[mixlab_protocol::Terminal], outputs: &[mixlab_protocol::Terminal]) {
```

Comment:
I got to this point and realised `Terminal` name conflicts with the component. It seemed to resolve to the right one here but I didn't like that it was ambiguous. I have just qualified it here but we could perhaps discuss clearer names.

Reply:
I'm not too opposed to these two types sharing a name given they represent the same concept, just in different domains. Prefixing the crate name adds clarity too. I think this is fine for now 👍